src/MobileUI/ViewModels/LeaderBoardViewModel.cs

```diff
@@ -1,18 +1,21 @@
 ﻿using System.Collections.ObjectModel;
 using System.Windows.Input;
+using CommunityToolkit.Mvvm.ComponentModel;
 using CommunityToolkit.Mvvm.Messaging;
 using CommunityToolkit.Mvvm.Input;
 using SSW.Rewards.Mobile.Controls;
 using SSW.Rewards.Mobile.Messages;
 
 namespace SSW.Rewards.Mobile.ViewModels;
 
-public class LeaderBoardViewModel : BaseViewModel, IRecipient<PointsAwardedMessage>
+public partial class LeaderBoardViewModel : BaseViewModel, IRecipient<PointsAwardedMessage>
 {
     private ILeaderService _leaderService;
     private IUserService _userService;
     private bool _loaded;
-    private ObservableCollection<LeaderViewModel> searchResults = new ();
+    
+    [ObservableProperty]
+    private ObservableCollection<LeaderViewModel> _searchResults = new ();
```

Comment:
This doesn't need to be an observable property as it's an ObservableCollection which already raises a property changed notification when items are added or removed.

But with that said, it's also private so not sure what the point is. This could probably be a List instead (and probably should).

Reply:
Ok I've now seen the public one below too. I think revert the change on these for now and let's reveiw properly.

Reply:
Done - these have been reverted for now.